core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -515,7 +521,10 @@ class TransactionStateManager(brokerId: Int,
    * metadata cache with the transactional ids. This operation must be resilient to any partial state left off from
    * the previous loading / unloading operation.
    */
-  def loadTransactionsForTxnTopicPartition(partitionId: Int, coordinatorEpoch: Int, sendTxnMarkers: SendTxnMarkersCallback): Unit = {
+  def loadTransactionsForTxnTopicPartition(partitionId: Int,
+                                           coordinatorEpoch: Int,
+                                           sendTxnMarkers: SendTxnMarkersCallback,
+                                           hadTransactionStateAlreadyLoaded: Boolean): Unit = {
```

Comment:
```suggestion
                                           hasTransactionStateAlreadyLoaded: Boolean): Unit = {
```

Reply:
I actually had it this way before, but we load in this method (and check afterwards) so I thought "had" conveyed that this happened before the method. I will think a bit more about this because there may be a better way to phrase.

Reply:
Ohhh, I see, maybe hasTransactionStateBeenLoadedBefore?

Reply:
yeah. Something like that. The key is that it is currently loaded when we make the call. Because it's ok to load in the past and unload. We want to make sure we clarify that is is loaded and running. Maybe even like transactionCoordinatorAlreadyRunning.

Reply:
I opted to go for simpler and chose `transactionStateLoaded`. I updated the method to say the same.